pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java

```diff
@@ -881,4 +963,98 @@ private void setupOutput(ContextImpl contextImpl) throws Exception {
             Thread.currentThread().setContextClassLoader(this.instanceClassLoader);
         }
     }
+
+    private static <T> Schema<T> getSinkSchema(Record<?> record, Class<T> clazz) {
```

Comment:
we could move these 3 methods to some other class that does similar things, like TopicSchema.
but I have no strong opinion

Reply:
We're doing these methods bc we don't have a topic. So it feels weird to put these in a class named `TopicSchema`